packages/graphqlsp/src/quickInfo.ts

```diff
@@ -87,16 +90,16 @@ export function getGraphQLQuickInfo(
     );
 
     return {
-      kind: ts.ScriptElementKind.string,
+      kind: ts.ScriptElementKind.label,
       textSpan: {
         start: cursorPosition,
         length: 1,
       },
-      kindModifiers: '',
-      displayParts: Array.isArray(hoverInfo)
-        ? hoverInfo.map(item => ({ kind: '', text: item as string }))
-        : [{ kind: '', text: hoverInfo as string }],
-    };
+      kindModifiers: 'text',
+      documentation: Array.isArray(hoverInfo)
```

Comment:
I'm admittedly a bit confused about this. It looks like `displayString` may be reserved for code-like references, while `documentation` is only for "meta" information?

This could correspond to the hover string, where `displayString` might be displayed in monospace, while `documentation` is the smaller sans-serif text below it, if I'm not mistaken